### PR TITLE
GTX-455 - Implement ability to run tests in parallel

### PR DESCRIPTION
**Pull request by cma06**

### CODE REVIEW

File: gameClientToGameWindowTests/errorMessage.spec.ts [hunk 1]
```diff
@@ -47,6 +51,8 @@ readGames().forEach((game: string) => {
         });
 
         test('reality check', async () => {
+            // set a 3 minute timeout for this test
+            test.setTimeout(300000);
```
Comment (gwenso) [inline gameClientToGameWindowTests/errorMessage.spec.ts]: minor suggestion: use underscores like the other timestamps in the code for consistency

File: gameClientToGameWindowTests/errorMessage.spec.ts [hunk 2]
```diff
@@ -60,16 +66,14 @@ readGames().forEach((game: string) => {
 
             await spin(page, consoleMessages);
 
-            // Wait 61s so the reality check timer kicks in
-            await page.waitForTimeout(61000);
+            // Wait 121s so the reality check timer kicks in
+            await page.waitForTimeout(121000);
```
Comment (gwenso) [inline gameClientToGameWindowTests/errorMessage.spec.ts]: same suggestion here, underscores for times

File: gameClientToGameWindowTests/errorMessage.spec.ts (hunk 1 above)
Comment (cma06) [reply gameClientToGameWindowTests/errorMessage.spec.ts]: sure, done that

File: gameClientToGameWindowTests/errorMessage.spec.ts (hunk 2 above)
Comment (cma06) [reply gameClientToGameWindowTests/errorMessage.spec.ts]: sure, done that

File: types/userAccount.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export interface Account {
```
Comment (abindukalpa) [inline types/userAccount.ts:1]: Any reason for this being an interface?

Comment (cma06) [reply types/userAccount.ts:1]: I actually wanted it to be a type, but the linter keeps making it an interface before i commit

**Review by MattClare9743 — APPROVED** [review]

**Pull request merged**